### PR TITLE
chore(core): add more logging at server shutdown and wal apply failure

### PR DESCRIPTION
**Pull request by ideoma**
Adds logs about QuestDB shutdown.
Adds details of the seqTxn of the suspended WAL table error log.

### CODE REVIEW

**Review by bluestreak01 — APPROVED** [review]

File: core/src/main/java/io/questdb/ServerMain.java [hunk 1]
```diff
@@ -271,17 +277,19 @@ public synchronized void start(boolean addShutdownHook) {
     private void addShutdownHook() {
         Runtime.getRuntime().addShutdownHook(new Thread(() -> {
             try {
+                System.err.println("SIGTERM received");
```
Comment (jerrinot) [inline core/src/main/java/io/questdb/ServerMain.java:280]: why is it twice? it's the same above.

Comment (jerrinot) [reply core/src/main/java/io/questdb/ServerMain.java:280]: ignore me, it's to out vs err

Comment (bluestreak01) [reply core/src/main/java/io/questdb/ServerMain.java:280]: one for STDERR and another for SDTOUT

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 15 / 19 (78.95%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/ServerMain.java|4|7|57.14%|
|:large_blue_circle:|io/questdb/cairo/wal/WalTxnDetails.java|7|8|87.50%|
|:large_blue_circle:|io/questdb/cairo/wal/ApplyWal2TableJob.java|4|4|100.00%|

File: core/src/main/java/io/questdb/ServerMain.java (hunk 1 above)
Comment (jerrinot) [reply core/src/main/java/io/questdb/ServerMain.java:280]: yeah, my eyes failed me when I looked at this for the first time.

**Pull request merged**